src/main/resources/META-INF/plugin.xml

```diff
@@ -1,8 +1,7 @@
 <idea-plugin url="https://artemis.ase.in.tum.de/">
     <id>de.tum.www1.orion</id>
     <name>Orion - Artemis Programming Exercise Integration</name>
-    <vendor email="alexander.ungar@tum.de" url="https://artemis.ase.in.tum.de">Technical University of Munich</vendor>
-    <description><![CDATA[
+    <vendor email="alexander.ungar@tum.de" url="https://artemis.ase.in.tum.de">Technical University of Munich</vendor>    <description><![CDATA[
```

Comment:
Is this change intended? Looks off to me with description being in the same line

Reply:
no it wasn't :D